content/docs/1.10.0/nodes-and-volumes/nodes/scheduling.md

```diff
@@ -27,15 +27,83 @@ At this time, if `Replica Node Level Soft Anti-Affinity` is un-checked and `Repl
+  - `Storage Minimal Available Percentage`: 10%
+  - Minimum required available space: `(Storage Maximum × Storage Minimal Available Percentage) / 100` ➔`(8 GB × 10) / 100 = 0.8 GB`
+  - Disk Y pass the **Actual Space Usage Condition**, Since remaining available space (1.5 GB) is greater than minimum required available space (0.8 GB).
+
+  Then check if satisfies the **Scheduling Space Condition**
+  - Scheduled space: `2 GB`
+  - Total Scheduled space: `(Size + Storage Scheduled) ➔ (1 GB + 2 GB = 3 GB)`
+  - Storage Reserved: `1 GB`
+  - `Over Provisioning Percentage`: 100% (default)
+  - Max provisionable storage: 
+  `(Storage Maximum - Storage Reserved) × Over Provisioning Percentage / 100` ➔ `(8 GB - 1 GB) × 100 / 100 = 7 GB`
+  - Disk Y pass the **Scheduling Space Condition**, Since total scheulded space (3 GB) is less than max provisionable storage (7 GB), Disk Y is schedulable. If Disk Y matches the disk tag, Longhorn can schedule the replica on Disk Y.
+
+
+**Anti-Affinity Behavior**
+
+Let’s assume both Disk X and Disk Y pass the **Actual Space Usage Condition** and **Scheduling Space Condition**, and Disk X already hosts a replica. 
+
+
+Now consider the anti-affinity setting:
+
+- **Hard Anti-Affinity**
+    - Longhorn will schedule the new replica on Disk Y to avoid co-locating replicas. 
+    - If Disk Y is unsuitable (e.g., mismatched tags), scheduling fails—replicas cannot share the same disk under hard anti-affinity.
+
+
+- **Soft Anti-Affinity**
+    - The setting `Replica Disk Soft Anti-Affinity` is enabled.
+    - Longhorn prefers to schedule the new replica on Disk Y (if it meets tag and space requirements) to avoid co-locating replicas, even if Disk X is otherwise viable.
+    - If Disk Y is unsuitable (e.g., insufficient space or mismatched tags), Longhorn may still schedule on Disk X if it meets all other conditions, as soft anti-affinity allows sharing as a fallback.
+    
+    **Soft Anti-Affinity Levels**
+    -   **Replica Disk Level Soft Anti-Affinity**  
+    Allow scheduling on disks with existing healthy replicas of the same volume
+
+    -   **Replica Node Level Soft Anti-Affinity**  
+    Allow scheduling on nodes with existing healthy replicas of the same volume
+
+    -   **Replica Zone Level Soft Anti-Affinity**  
+    Allow scheduling on zones with existing healthy replicas of the same volume
 
-Once the node and zone stage is satisfied, Longhorn will decide whether it can schedule the replica on any disk of the node. Longhorn will check the available disks on the selected node with the matching tag, the total disk space, and the available disk space. It will also check whether another replica already exists and whether anti-affinity is set to be "hard" (no sharing) or "soft" (prefer not to share.)
-
-For example, after the node and zone stage, Longhorn finds `Node A` satisfies the requirements for scheduling a replica to the node. Longhorn will check all the available disks on this node.
```

Comment:
I think we can add these back and modify them a bit before introduce the following logic and examples.